src/docs/operator-guides/how-to-edit-pages/index.md

```diff
@@ -0,0 +1,295 @@
+    link should point to an internal folder within your marketplace,
+    with a partial URL.
+  - **External link** means that the link will open in a new tab. You
+    need to specify the complete URL.
+- Link text: The text within the call to action button. Try to make the
+  text short and simple to engage your visitors and incite them to
+  action.
+- Link address: The URL or folder for the button.
+  - **Internal link:** you have to specify a folder within your
+    marketplace. For example “/p/about”, it will take your users to
+    `https://yourmarketplaceurl.com/p/about`
+  - **External link:** you have to specify a complete URL. You need to
+    add http:// or https:// to your link.
+
+Adding a call to action to your Section is optional.
+
+### Specify Section appearance
+
+You can choose between Default or Custom.
+
+**Default**
+
+The default background is white or shaded white. The color alternates
+within each Section to make a slight yet almost imperceptible difference
+between them. Between rgb(252,252,252) and rgb(250,250,250)
+
+**Custom**
+
+With custom appearance, you can specify any other background color with
+the color picker or add a background image to a section. The image will
+always be set on top of the background color. Unless your image is
+transparent, it will cover the background color. Your image will be
+expanded to fit the dimensions of the section. You can also define the
+text color. The options are black and white. The background and the
+theme affect the whole section, including all the blocks contained in
+it.
+
+## Add blocks (Optional)
+
+Every Section can be expanded by adding Blocks. You can add Blocks by
+clicking on the “+ Add new Block…” link at the bottom of the section.
+
+![add new block link](/add-new-block.png 'add new block button')
+
+Each Block needs to contain a unique Block ID. All the other fields to
+create a block are optional, however, they are recommended.
+
+- Block ID: a unique identifier for each block within the section.
+  Similar to the section id. Without this ID, you cannot save changes to
+  your landing page.
+- Media: choose if your block features an image, Youtube video embed, or
+  no media.
+- Text content: Title content, title size and text content. These are
+  comparable to the text content features for the main section.
+- Button content: Button type, button text and button link. These are
+  comparable to the button content features for the main section.
+
+The blocks are the soul of each Section. Without block content, each
+Section will be indistinguishable from the others. Once you add blocks,
+each one will be displayed and ordered according to the type of Section.
+There is no limit to the number of blocks that you can add.
+
+You can even have Sections that don’t contain the basic information in
+the Section fields, and add only information within blocks.
+
+### Adding media to a Block
+
+You can add an image or a Youtube video as part of the content of every
```

Comment:
"add an image or embed a YouTube" video

It's good to clarify that images get uploaded but videos only embedded. Also, YouTube always with big T.